src/css/components/icon.scss

```diff
@@ -0,0 +1,15 @@
+
+.icon-container {
+  height: 3.125rem;
+  width: 3.9rem;
+}
+
+.icon {
+  max-height: 3.1rem;
+}
+
+.icon-alt {
+  .cls-1 {
```

Comment:
Yea it was automatically generated by illustrator.

I almost thought it would be easier to use that class for now rather then editing every single svg.

I think a better system eventually needs to be put in place for svg, but this will do for launch